examples/jsm/nodes/core/NodeBuilder.js

```diff
@@ -43,13 +43,14 @@ const toFloat = ( value ) => {
 
 class NodeBuilder {
 
-	constructor( object, renderer, parser ) {
+	constructor( object, renderer, parser, scene = null ) {
```

Comment:
Hmm... Using this approach, we can try soon create `sceneBackground`, it can change the code if `.background` is different, the same not happen with per frame approach.